sdk/webpubsub/azure-messaging-webpubsub-client/src/main/java/com/azure/messaging/webpubsub/client/implementation/websocket/WebSocketClientHandler.java

```diff
@@ -68,31 +83,27 @@ protected void channelRead0(ChannelHandlerContext ctx, Object msg) {
             return;
         }
 
-        if (msg instanceof FullHttpResponse) {
-            FullHttpResponse response = (FullHttpResponse) msg;
-            throw loggerReference.get()
-                .logExceptionAsError(new IllegalStateException("Unexpected FullHttpResponse (getStatus="
-                    + response.status() + ", content=" + response.content().toString(CharsetUtil.UTF_8) + ')'));
+        if (msg instanceof WebSocketFrame) {
+            channelRead0(ctx, (WebSocketFrame) msg);
+        } else {
+            loggerReference.get().atWarning().addKeyValue("messageType", msg).log("Unknown message type. Skipping.");
```

Comment:
The value of `addKeyValue` is an object? Can it be serialized to log?

Reply:
I wanted the type name serialiozed. Will be specific about it.